app/src/main/java/com/x8bit/bitwarden/data/auth/repository/util/RefreshTokenResponseExtensions.kt

```diff
@@ -24,7 +24,6 @@ fun RefreshTokenResponseJson.toUserStateJson(
             email = jwtTokenData.email,
             isEmailVerified = jwtTokenData.isEmailVerified,
             name = jwtTokenData.name,
-            hasPremium = jwtTokenData.hasPremium,
```

Comment:
Where does this data come from otherwise? The main sync response?

Reply:
Correct

Reply:
This was an intermittent issue I have seen where my account's premium status would just disappear. I finally tracked it to this thing that was coming back `false` even though I have premium. I double-checked with iOS they do not use this value.

Reply:
OK sounds good.